x/milestone/abci/abci.go

```diff
@@ -476,3 +479,7 @@ func ValidateMilestoneProposition(ctx sdk.Context, milestoneKeeper *keeper.Keepe
 
 	return nil
 }
+
+func ShouldErrorOnValidatorNotFound(ctx sdk.Context) bool {
+	return ctx.BlockHeight() >= helper.GetTallyFixHeight() || ctx.BlockHeight() < helper.GetDisableValSetCheckHeight()
```

Comment:
Is there a relationship between `GetTallyFixHeight` and `GetDisableValSetCheckHegith`? Should the former not be greater than the second and in fact, should it not be `GetDisableValSetCheckHegith` + 1?

Reply:
Yeah so `GetDisableValSetCheckHegith` would be the height we disabled the check and `GetTallyFixHeight` would be the height where it'd be re-enabed.

Reply:
Ah I see. And there should be an error on validator not found only before the height at which you had disabled this and once the proper validator counting starts happening. Makes sense.